src/plone/staticresources/profiles/default/registry/resources.xml

```diff
@@ -193,7 +193,8 @@
 
   <records prefix="plone.resources/moment"
             interface='Products.CMFPlone.interfaces.IResourceRegistry'>
-      <value key="js">++plone++static/components/moment/min/moment-with-locales.min.js</value>
+      <value key="js">++plone++static/components/moment/min/moment.min.js</value>
+      <value key="url">++plone++static/components/moment/locale</value>
   </records>
 
```

Comment:
I think this requires an upgrade step for beta2

Reply:
Agree, thanks for reminding!Sshould I start the `to_beta2` profile in p.a.upgrade?

Reply:
I would say so :)